docs/upgrade/xenial_upgrade_in_place.rst

```diff
@@ -1,31 +1,29 @@
-Ubuntu 16.04 LTS (Xenial) - Upgrading in Place
+Ubuntu 16.04 LTS (Xenial): upgrading in place
 ==============================================
```

Comment:
Consistent with the other section titles in the docs, I've already changed the "Backup, Install, Restore" doc to title case, I'd recommend leaving this in title case as well.